zipline/finance/slippage.py

```diff
@@ -255,15 +255,17 @@ def process_order(self, event, order):
         volume_share = min(total_volume / event.volume,
                            self.volume_limit)
 
-        simulated_impact = volume_share ** 2 \
-            * math.copysign(self.price_impact, order.direction) \
-            * event.price
+        # if this is a limit order, do not impact the fill price
+        if order.limit:
+            simulated_impact = 0
+        else:
+            simulated_impact = volume_share ** 2 \
+                * math.copysign(self.price_impact, order.direction) \
+                * event.price
```

Comment:
Would this impact TRAIL orders, or does order.limit exist IFF the order is an LMT order?